core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -221,14 +221,28 @@ public String toString() {
    * @return the {@link InodeDirectory} representation
    */
   public static InodeDirectory fromJournalEntry(InodeDirectoryEntry entry) {
-    Permission permission;
+    String owner;
+    String group;
+    int mode;
+    if (entry.hasOwner()) {
+      owner = entry.getOwner();
+    } else {
+      // remove in 2.0
```

Comment:
as discussed offline, I am fine with removing them right now as the entire security scheme is still in alpha